modules/tts/tts-utils/tts-adapter-google/src/test/java/org/daisy/pipeline/tts/google/impl/GoogleTTSSSMLTest.java

```diff
@@ -0,0 +1,137 @@
+package org.daisy.pipeline.tts.google.impl;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.Map;
+import java.util.TreeMap;
+
+import net.sf.saxon.s9api.Axis;
+import net.sf.saxon.s9api.Processor;
+import net.sf.saxon.s9api.QName;
+import net.sf.saxon.s9api.SaxonApiException;
+import net.sf.saxon.s9api.XdmNode;
+
+import org.custommonkey.xmlunit.Diff;
+import org.daisy.common.xslt.ThreadUnsafeXslTransformer;
+import org.daisy.common.xslt.XslTransformCompiler;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.xml.sax.SAXException;
+
+import com.xmlcalabash.util.TreeWriter;
+
+public class GoogleTTSSSMLTest {
```

Comment:
Not sure what to do with this file. It hasn't been touched after it was copied from the espeak project.

Reply:
I was thinking of keeping it in case of modifications of the transform to ensure nothing breaks (if modifications are required) when building the module independently.

If you think this part is not necessary, it can be deleted.

Reply:
Well it could be useful of course: any test could possibly flag a regression. But I'd rather have tests that were added for a specific reason. This test really just seemed like a copy, and in general I don't like copying stuff because that way you potentially copy bugs and bad code. (This test in particular I don't like so much because it makes use of the Saxon and Calabash APIs, while I'd like to become independent of those APIs as much as possible.)